CHANGE-NOTES.md

```diff
@@ -3,6 +3,7 @@
 ## v3.6.1
 - Fixed: notification about unmanaged current branch wasn't showing up.
 - Fixed: in case of red edge from a branch to its parent, the unique history of a branch can no longer include any commits reachable from its parent.
+- Improved: side-effecting actions (like Rebase, Pull, Discover etc.) are disabled while another side-effecting action is ongoing.
```

Comment:
in fact, does it not make this a minor change? 😄

Reply:
Huh... as usual, with end-user software where the only consumer is a hooman (and not another service/script etc.) the version numbering is relatively minor importance. The most important (if at all) part is tracking IntelliJ compatibility. We've assumed the convention of using minor version bumps to signify new features added as well... still, I'd say it doesn't really constitute a new feature, it's more of UX fix to prevent the users from putting the plugin/git repo in a messy state